hack/test-cmd.sh

```diff
@@ -2031,7 +2031,7 @@ __EOF__
     fi
     kubectl describe -f "${file}" "${kube_flags[@]}"
     # Command
-    kubectl replace -f $replace_file --force "${kube_flags[@]}"
+    kubectl replace -f $replace_file --force --cascade "${kube_flags[@]}"
```

Comment:
Without `--cascade`, the controlled pods are not deleted. They haven't caused any test failure only because the RCs created in later tests happen to have the same selector, so they select the left-over pods and deletes them. We won't be so lucky after introducing controller ref, because RC created later will see these left-over pods having a controller (which is deleted) and won't try to adopt them, so these pods will exist forever.